src/components/OptionsRow.js

```diff
@@ -51,7 +51,7 @@ function OptionsRow({
 */
   return (
     <Container fluid>
-      <Row className="pt-4 opt-row">
+      <Row className="pt-4 mb-4">
```

Comment:
This now also adds extra space when there is a warning. Is there a way where we have a consisten amount of space between the options row and the figure rows? I imagine the warning box has margin on top that we can delete?